templates/newsletter/render_form.html.twig

```diff
@@ -0,0 +1 @@
+{{ render(controller('AppBundle\\Controller\\EnMarche\\NewsletterController::renderNewsletterFormAction', { options: options|default({ }) })) }}
```

Comment:
juste `controller('Controller::action', options|default({ }))` no?

Reply:
no

Reply:
Can you be more specific ?

Reply:
I have to specify the argument key to say thats all options are in the argument controller options.